### DISCUSSION TITLE
Continuous build  46c8834

### DISCUSSION THREAD

**Discussion by Terracotta-6** [artisan-roaster-scope/artisan#1304] [General]
kaleido roaster What causes the burner control delay?
![f4ca8f4bf837ed8e9b4c803a950c2fa](https://github.com/artisan-roaster-scope/artisan/assets/149906183/c066826b-e86c-43bc-93bb-c5c715769a14)

**Comment by Terracotta-6** [comment]
Below is a past version.   2.8.4 Version  ac17fbe 
![e571ae95074dff428615022855382ef](https://github.com/artisan-roaster-scope/artisan/assets/149906183/9b5fa500-185a-4f52-acce-9c51de96958f)

**Comment by MAKOMO** [comment]
Which commit is the past version? Which Kalaido setup are you using here?

**Reply by MAKOMO** [reply]
Please share profiles and settings and other details relevant.

**Reply by Terracotta-6** [reply]
Kaleido Legacy TC4 communication protocol .
Bake the new coffee once tomorrow and record the profiles & settings upload.
Always used the same configuration, only after upgrading the software version, coffee roasting records found a burner delay issue.
Version 46c8834 Loading Version ac17fbe records are displayed normally.

**Reply by MAKOMO** [reply]
Could you please attach your settings and this profile as .alog file? Thanks!

**Comment by Terracotta-6** [comment]
Restore factory Settings and show normal burner curve. Sorry to bother you!
[23-11-21_1628.alog.txt](https://github.com/artisan-roaster-scope/artisan/files/13423903/23-11-21_1628.alog.txt)
[artisan-settings.aset.txt](https://github.com/artisan-roaster-scope/artisan/files/13423907/artisan-settings.aset.txt)

**Reply by MAKOMO** [reply]
Ok, so I understand that this one is resolved.

**Comment by Terracotta-6** [comment]


**Reply by MAKOMO** [reply]
Not too good to reuse one thread for discussing more than one issue. I am not sure what you refer to as "robust curve". You activated the RoR projection lines (introduced in v2.6.0) which are by default deactivated. They are based on the RoRoR and are thus way more noisy then the ET/BT curve projections.  See the [Release Notes of v2.6.0](https://artisan-roasterscope.blogspot.com/2022/03/artisan-v260.html).
